src/styles/colours/_panels.scss

```diff
@@ -162,6 +163,24 @@
   .item-#{$i} {
     animation: fadeIn 0.3s ease-in ($i * 0.03s) forwards;
     opacity: 0;
+
+    .item-#{$i}--remove {
+      background: url('../assets/img/close.svg') no-repeat center center / 0.5rem 0.5rem;
```

Comment:
If possible, refactor to have a common class for the close icon stuff (since it's the same in the saved items).